lib/turn.rb

```diff
@@ -0,0 +1,21 @@
+class Turn
+	attr_reader :guess, :card
+
+	def initialize(guess, card)
+		@guess = guess
+		@card = card
+	end
+
+	def correct?
+		@guess == card.answer
```

Comment:
instead of calling to `card`, it's more conventional to call to `@card` in this scenario. Since `card` is referencing your attr_reader, which returns `@card`, you might as well call directly to `@card` and cut out the middle man.